sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/SearchIndexStatistics.java

```diff
@@ -33,17 +33,19 @@ public final class SearchIndexStatistics implements JsonSerializable<SearchIndex
     /*
      * The amount of memory in bytes consumed by vectors in the index.
      */
-    private Long vectorIndexSize;
+    private final long vectorIndexSize;
 
     /**
      * Creates an instance of SearchIndexStatistics class.
      *
      * @param documentCount the documentCount value to set.
      * @param storageSize the storageSize value to set.
+     * @param vectorIndexSize the vectorIndexSize value to set.
      */
-    public SearchIndexStatistics(long documentCount, long storageSize) {
+    public SearchIndexStatistics(long documentCount, long storageSize, long vectorIndexSize) {
```

Comment:
If this type has been GA'd before we'll need to remove `vectorIndexSize` from the required properties with a Swagger transform as this adds a new parameter to the constructor.